README.md

```diff
@@ -1,68 +1,333 @@
+
+1. Download and install [`NodeJS`](https://nodejs.org/en/) if your system doesn't have it yet.
+2. Setup the [React Native environment](https://reactnative.dev/docs/environment-setup) in your system.
+3. Open a terminal and enter the commands below in your project path.
+```bash
+$ cd react-native-redux-sample/ReactNativeWithSendBird
+$ npm install
+```
+4. Run the sample app by entering the commands below.
+```bash
+$ react-native run-ios
+$ react-native run-android
+```
+
+Before entering the two commands above, you should run the simulator from the [Android Studio](https://developer.android.com/studio/) in advance. The command for iOS works without further action.
+
+### Here are the steps to install Chat SDK
+
+Follow the simple steps below to build the Chat SDK into your client app.
+
+#### Step 1: Create a Sendbird application from your dashboard
+
+A Sendbird application comprises everything required in a chat service including users, message, and channels. To create an application:
+
+1. Go to the [Sendbird Dashboard](https://dashboard.sendbird.com/auth/signup) and enter your email and password, and create a new account. You can also sign up with a Google account.
+2. When prompted by the setup wizard, enter your organization information to manage Sendbird applications.
+3. Lastly, when your dashboard home appears after completing setup, click **Create +** at the top-right corner.
+
+Regardless of the platform, only one Sendbird application can be integrated per app; however, the application supports communication across allSendbird’s provided platforms without any additional setup. Sendbird currently supports iOS, Android, web, .NET, and Unity client apps.
+
+> Note: All the data is limited to the scope of a single application, thus the users in different Sendbird applications are unable to chat with each other.
+
+#### Step 2: Install the Chat SDK
+
+If you’re familiar with using external libraries or SDKs, installing the Chat SDK is simple.You can install the Chat SDK with package manager npm or Bower by entering the command below on the command line.
+
+- **Npm**
+
+> Note: To use npm to install the Chat SDK, Node.js must be first installed on your system.
+
+```bash
+$ npm install sendbird (request to npm server)
+```
+
+Install via `Npm` and import like below in your `TypeScript` file. 
+
+```bash
 import * as SendBird from "sendbird";
 var sb = new SendBird({ appId: "APP_ID" });
 // do something...
 ```
 
-If you have trouble importing `Sendbird`, please check your `tsconfig.json` file and change the value of `"allowSyntheticDefaultImports"` to `true` in `compilerOptions`.
+If you have trouble importing Sendbird, please check your `tsconfig.json` file and change the value of `allowSyntheticDefaultImports` to true in `compilerOptions`.
+
+- **Bower**
```

Comment:
`bower` is quite old-fashioned way to install a package. How about adding `yarn add sendbird` instead?

Reply:
Done!